kafka-broker/rc.local

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
shell-based startup scripts are pretty much a thing of the past.  Could you try replacing this with a couple of systemd unit files instead please (one for zookeeper and another one for kafka-server which should depend on the zookeeper one).

Reply:
rc.local has been removed, a systemd hybrid approach is in place on this first iteration.